src/rcx/src/ext-py.i

```diff
@@ -40,7 +40,11 @@ using namespace odb;
 using rcx::Ext;
 %}
 
+%include <std_string.i>
 %include "../../Exception-py.i"
+%import "odb/odb.h"
+%feature ("flatnested");
```

Comment:
Could you document why you added this flag?

Reply:
done